packages/wine-staging-git/wine-staging-git.pacscript

```diff
@@ -70,24 +70,31 @@ build() {
 
 install() {
   cd "wine32-build"
-  mkdir -p "${pkgdir}/usr/lib32/wine"
-  make prefix="${pkgdir}/usr" \
+  sudo mkdir -p "${pkgdir}/usr/lib32/wine"
+  sudo make prefix="${pkgdir}/usr" \
     libdir="${pkgdir}/usr/lib32" \
     dlldir="${pkgdir}/usr/lib32/wine" \
     install
   cd "../wine64-build"
-  mkdir -p "${pkgdir}/usr/lib/wine"
-  make prefix="${pkgdir}/usr" \
+  sudo mkdir -p "${pkgdir}/usr/lib/wine"
+  sudo make prefix="${pkgdir}/usr" \
     libdir="${pkgdir}/usr/lib" \
     dlldir="${pkgdir}/usr/lib/wine" \
     install
 
-  install -d -m755 "${pkgdir}/usr/share/fontconfig/conf.default"
-  install -D -m644 "${SRCDIR}/${name}/30-win32-aliases.conf" -t "${pkgdir}/usr/share/fontconfig/conf.avail"
-  ln -s "../conf.avail/30-win32-aliases.conf" "${pkgdir}/usr/share/fontconfig/conf.default/30-win32-aliases.conf"
+  sudo install -d -m755 "${pkgdir}/usr/share/fontconfig/conf.default"
+  sudo install -D -m644 "${SRCDIR}/${name}/30-win32-aliases.conf" -t "${pkgdir}/usr/share/fontconfig/conf.avail"
 
-  install -D -m644 "${SRCDIR}/${name}/wine-binfmt.conf" "${pkgdir}/usr/lib/binfmt.d/wine.conf"
+  sudo install -D -m644 "${SRCDIR}/${name}/wine-binfmt.conf" "${pkgdir}/usr/lib/binfmt.d/wine.conf"
 
-  i686-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib32/wine/i386-windows"/*.dll
-  x86_64-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib/wine/x86_64-windows"/*.dll
+  sudo i686-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib32/wine/i386-windows"/*.dll
+  sudo x86_64-w64-mingw32-strip --strip-unneeded "${pkgdir}/usr/lib/wine/x86_64-windows"/*.dll
+}
+
+postinst() {
+  sudo ln -s "../conf.avail/30-win32-aliases.conf" "${pkgdir}/usr/share/fontconfig/conf.default/30-win32-aliases.conf"
```

Comment:
postinst is run from `/` so `../` can't exist. Figure out where this file actually installs to